src/beam/cli/main.py

```diff
@@ -1,10 +1,17 @@
 import os
+from dataclasses import dataclass
 from pathlib import Path
 
+from beta9 import config
 from beta9.cli.main import load_cli
-from beta9.config import SDKSettings
 
-from . import configure, login, quickstart
+from . import configure, login, logs, quickstart
+
+
+@dataclass
+class SDKSettings(config.SDKSettings):
```

Comment:
didn't we have this defined somewhere else as well? forgot how we set this up for the gateway/app URL

Reply:
Yep, good catch. Just added it here. https://github.com/beam-cloud/beta9/pull/336